src/collections/Form/index.d.ts

```diff
@@ -173,7 +173,7 @@ interface FormGroupProps {
 
 export const FormGroup: React.ComponentClass<FormGroupProps>;
 
-interface FormInputProps extends InputProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
+interface FormInputProps extends InputBaseProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
```

Comment:
How about adding   
`onChange`, `onInput`, `onSubmit` methods to the `InputProps` and removing `ReactFormEvents`? (If these methods are available)